server/asset/eth/eth.go

```diff
@@ -275,16 +276,35 @@ func NewBackend(configPath string, log dex.Logger, net dex.Network) (*ETHBackend
 	}
 	defer file.Close()
 
-	var endpoints []string
+	var endpoints []endpoint
 	endpointsMap := make(map[string]bool) // to avoid duplicates
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
-		line := strings.Trim(scanner.Text(), " ")
-		if line == "" || strings.HasPrefix(line, "#") || strings.HasPrefix(line, ";") || endpointsMap[line] {
+		line := strings.TrimSpace(scanner.Text())
+		if line == "" || strings.HasPrefix(line, "#") || strings.HasPrefix(line, ";") {
+			continue
+		}
+		parts := strings.Split(line, ",")
+		if len(parts) < 1 || len(parts) > 2 {
+			return nil, fmt.Errorf("invalid eth config line %q", line)
+		}
+		url := strings.TrimSpace(parts[0])
+		var priority uint16
+		if len(parts) == 2 {
+			priority64, err := strconv.ParseUint(strings.TrimSpace(parts[1]), 10, 16)
```

Comment:
`strconv.ParseUint: parsing "226135": value out of range`
Borderline clear enough, but maybe have the error say the expected range or max.

Reply:
Or allow mega numbers?